projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 13.6-a.1 - 2024-06-10
+### Bug fixes
+- External media: Ensured the connect URL has the correct blog ID and verification values. [#37689]
+- Slideshow: Ensure whole block is selectable in the editor [#37683]
+
+### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
+- Add onboarding_segment to site endpoint. [#37751]
+- Add placement on Proofread tracking. [#37763]
+- Add support for custom behavior on AI Inline Extensions. [#37589]
+- Admin menu: add link to Customizer in sites with block-based theme and custom CSS. [#37761]
+- AI Featured Image: let the backend decide the model for the image generation. [#37668]
+- Calypsoify: Deprecating functions and the class. [#37453]
+- Change codebase to use clsx instead of classnames. [#37708]
+- Classic Theme Helper: Move code from module to package. [#37260]
+- Comics CPT: add deprecation warning. [#37605]
+- External Media: Remove unneeded return value. [#37703]
+- Fix scroll on typing inside inline extensions. [#37753]
+- Jetpack AI: expose the upgrade URL on the feature endpoint, considering current plan and site type. [#37721]
+- Memberships: fixes paid content access in some cases of mixed products. [#37527]
+- Move ai form inline extensions version to production. [#37754]
+- Move AI Form to inline extension. [#37589]
+- Payments Block: localize block variations. [#37661]
+- Random Redirect: add deprecation warning. [#37609]
+- Subscribe overlay: use home URL in skip link. [#37711]
+- Updated package dependencies. [#37669] [#37706]
+- Updated recommendation links for Social. [#37693]
+- Updated recommendations section links for Social. [#37678]
+- Use correct const to check for inline extensions availability. [#37758]
+- Wordpress.com: Make jetpack scan link visible on atomic sites that defaults to wp-admin. [#37724]
+- WPCOM Block Editor: Deprecated functionality and moved it to mu-wpcom. [#37324]
```

Comment:
We had a Comics CPT?

```suggestion
## 13.6-a.1 - 2024-06-10
### Bug fixes
- External media: Ensure connect URL has the correct blog ID and verification values. [#37689]
- Slideshow: Ensure whole block is selectable in the editor. [#37683]

### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
- Add onboarding_segment to site endpoint. [#37751]
- Add placement on Proofread tracking. [#37763]
- Add support for custom behavior on AI Inline Extensions. [#37589]
- Admin menu: Add link to Customizer in sites with block-based theme and custom CSS. [#37761]
- AI Featured Image: Let the backend decide the model for the image generation. [#37668]
- Calypsoify: Deprecating functions and the class. [#37453]
- Change codebase to use clsx instead of classnames. [#37708]
- Classic Theme Helper: Move code from module to package. [#37260]
- Comics CPT: Add deprecation warning. [#37605]
- External Media: Remove unneeded return value. [#37703]
- Fix scroll on typing inside inline extensions. [#37753]
- Jetpack AI: Expose the upgrade URL on the feature endpoint, considering current plan and site type. [#37721]
- Memberships: Fix paid content access in some cases of mixed products. [#37527]
- Move AI Form inline extensions version to production. [#37754]
- Move AI Form to inline extension. [#37589]
- Payments Block: Localize block variations. [#37661]
- Random Redirect: Add deprecation warning. [#37609]
- Subscribe overlay: Use home URL in skip link. [#37711]
- Updated package dependencies. [#37669] [#37706]
- Updated recommendation links for Social. [#37693]
- Updated recommendations section links for Social. [#37678]
- Use correct const to check for inline extensions availability. [#37758]
- Wordpress.com: Make Jetpack Scan link visible on atomic sites that defaults to wp-admin. [#37724]
- WPCOM Block Editor: Deprecated functionality and moved it to mu-wpcom. [#37324]
```